src/main/java/taxi/controller/LogoutController.java

```diff
@@ -0,0 +1,16 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getSession().invalidate();
+        resp.sendRedirect("/login");
```

Comment:
lets add contextPath to all sendRedirect

Reply:
Done, thanks!

Reply:
Not done( 
Be attentive
![image](https://github.com/mate-academy/jv-web-security/assets/20326858/41457aeb-8c31-48d5-81a2-61067c6ce57e)